docs/uml/protocol/ExtEvSimulationSequence.puml

```diff
@@ -43,7 +43,7 @@ Scheduler -> ExtEvDataService: <font color=red>!</font> Activation(t1)
 deactivate Scheduler
 activate ExtEvDataService
 
-ExtSimulation -> ExtSimAdapter: <font color=red>!</font> CompletionMessage(newTriggers)
+ExtSimulation -> ExtSimAdapter: <font color=red>!</font> Completion(newTriggers)
```

Comment:
Sorry, this should still be `newTick`

Reply:
So, this would be still `CompletionMessage(newTick)` (below as well), since this is the other completion message from simonaAPI

Reply:
Ah sorry I must have forgotten to change that

Reply:
@marvinheintze Sorry, should still be `CompletionMessage` as I said above

Reply:
Oh you're right, sorry. All the Completions in this Diagram are CompletionMessages, right?

Reply:
No, sorry, only the ones `ExtSimulation -> ExtSimAdapter`